src/index.ts

```diff
@@ -0,0 +1,54 @@
+const round = (number: number) => Number(number.toFixed(2))
```

Comment:
I wonder what's the rationale of reducing accuracy through `round`?

Reply:
the thing is, you already have the possibility to choose between seconds, milliseconds, microseconds and nanoseconds. 

therefore it doesn't make sense to return e.g. seconds with arbitrary accuracy since you're very probably not interested in the nanoseconds part of the measurement if you choose to get seconds.

Reply:
I’m trying to implement a TCP CUBIC-like algorithm (for an overlay network that runs above UDP). The CUBIC algorithm natively uses “seconds” as the time unit, but it does need as much precision as possible. It would be most natural to request “seconds” unit from this library, instead of requesting “nanoseconds” and dividing by 1e9.
Would you be open to remove this `round` function, either globally or via an option like `.seconds({ round: false })`?

Reply:
i'm with @seriousManual here. if this is something you need i would recommend creating your own wrapper/adapter. btw, this rounding is not new functionality as part of the PR. it was already implemented, just in rather roundabout manner.

Reply:
yes, it seems to me like a very specific usecase, I want to keep the scope of this module as broad as possible.